GPU/Common/SplineCommon.cpp

```diff
@@ -223,6 +223,39 @@ static void spline_knot(int n, int type, float *knot) {
 	}
 }
 
+static void TessellateSplinePatchHardware(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch) {
+	SimpleVertex *&vertices = (SimpleVertex*&)dest;
+
+	for (int tile_v = 0; tile_v < spatch.tess_v + 1; ++tile_v) {
```

Comment:
I think this needs a comment to explain that it's generating simple input vertices for the spline-computing vertex shader.